.gitignore

```diff
@@ -47,6 +47,16 @@
 *.tar
 *.xz
 *.zst
+*.cmd
+*.d
+*.S
+*.config
```

Comment:
there are files `*d`, `*.S` and `*.config` tracked by git.

Reply:
![image](https://github.com/armbian/linux-rockchip/assets/60068659/27e91cc3-e98b-4ed0-a03c-f4445d06e61a)
is not use .gitignore file ?

Reply:
test.d is a dir: https://github.com/armbian/linux-rockchip/tree/rk-5.10-rkr5.1/tools/testing/selftests/ftrace/test.d
There are already *.S in git: https://github.com/armbian/linux-rockchip/blob/rk-5.10-rkr5.1/usr/initramfs_data.S
Also *.config: https://github.com/armbian/linux-rockchip/blob/rk-5.10-rkr5.1/tools/testing/selftests/wireguard/qemu/kernel.config